src/Compilers/CSharp/Portable/Symbols/Source/SourceConstructorSymbol.cs

```diff
@@ -225,9 +229,7 @@ protected override IAttributeTargetSymbol AttributeOwner
         }
 
         internal override bool IsExpressionBodied
-        {
-            get { return false; }
-        }
+            => _isExpressionBodied;
```

Comment:
extended to full bodied get accessor. aren't you going to change all get-only properties to expression bodies in the near future? Thought I read it somewhere.